linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_conf_all_accept_local/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Disable Accepting Packets Routed Between Local Interfaces'
+
+description: '{{{ describe_sysctl_option_value(sysctl="net.ipv4.conf.all.accept_local", value="0") }}}'
+
+rationale: |-
+    Configure <tt>net.ipv4.conf.all.accept_local=0</tt> to consider as invalid the packets
+    received from outside whose source is the 127/8 network.
```

Comment:
I'm not sure if "127/8 network" is a widely understood term. I would instead say 127.0.0.0/8 address block.

Reply:
```suggestion
    received from outside whose source is the 127.0.0.0/8 address block.
```